packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/triggers/feature_flags/v0/mod.rs

```diff
@@ -66,21 +62,24 @@ pub fn create_feature_flag_data_trigger(
     let latest_block_height = context.platform.state.height();
 
     if enable_at_height < latest_block_height {
-        let err = create_error(
-            context,
-            document_create_transition,
+        let err = DataTriggerConditionError::new(
+            context.data_contract.id(),
+            document_transition.base().id(),
             "This identity can't activate selected feature flag".to_string(),
         );
+
         result.add_error(err);
+
         return Ok(result);
     }
 
-    if context.owner_id != top_level_identity {
-        let err = create_error(
-            context,
-            document_create_transition,
-            "This Identity can't activate selected feature flag".to_string(),
+    if context.owner_id != feature_flags_contract::OWNER_ID {
```

Comment:
I think you are introducing a bug here. The top level identity isn't the same as the person who registered the feature flags contract. Or is it?

Reply:
It is